src/lib/cache/subCache.cpp

```diff
@@ -734,6 +737,8 @@ void subCacheItemInsert
   cSubP->expression.geometry   = geometry;
   cSubP->expression.coords     = coords;
   cSubP->expression.georel     = georel;
+  cSubP->httpInfo.url          = url.c_str();
+  cSubP->httpInfo.verb         = POST;  // FIXME PR: Unhardwire
```

Comment:
Fixed in another branch, merged to develop then merged into this branch, and into its 'parent'